k2/csrc/algorithms.h

```diff
@@ -22,6 +22,10 @@ namespace k2 {
 class Renumbering {
  public:
   Renumbering() = default;
+  // copy constructor
+  Renumbering(const Renumbering &src) = default;
+  // move constructor
+  Renumbering(Renumbering &&src) = default;
```

Comment:
Move assignment operator is also needed if move constructor is present.

```cpp
Renumbering &operator=(Renumbering &&) = default;

Reply:
ok..